lib/fizzy/execute.hpp

```diff
@@ -64,10 +64,23 @@ struct Instance
     std::vector<uint64_t> globals;
     std::vector<ExternalFunction> imported_functions;
     std::vector<ExternalGlobal> imported_globals;
+
+    Instance(Module _module, bytes_ptr _memory, size_t _memory_max_pages, table_ptr _table,
```

Comment:
`explicit`?

Reply:
`explicit` is useful only for single parameter constructors (or the ones where parameters after the first one have default values), others can only be called explicitly anyway.

Reply:
Ok, C++ noob here.